cirq/devices/grid_qubit_test.py

```diff
@@ -32,6 +32,36 @@ def test_xmon_qubit_eq():
     eq.make_equality_group(lambda: cirq.GridQubit(50, 25))
 
 
+def test_square():
+    assert cirq.GridQubit.square(2, top=1) == [cirq.GridQubit(1, 0),
```

Comment:
Doesn't confirm left works. Doesn't confirm top has default.

Reply:
Now tests default case for both and specified case for top, left.